langchain/src/vectorstores/similarity-score-threshold.ts

```diff
@@ -0,0 +1,72 @@
+import { VectorStore, VectorStoreRetriever } from "./base.js";
```

Comment:
Filename convention for `src` files is uses underscores:
`similarity_score_threshold.ts`

Reply:
Done :)